### PR TITLE
Automated clusterGroupSelector Fleet-32 test

### PR DESCRIPTION
**Pull request by sbulage**
Fixes: #235

### CODE REVIEW

**Comment by sbulage** [comment]
CI Run for 2.10.0: https://github.com/rancher/fleet-e2e/actions/runs/11891682293

**Comment by sbulage** [comment]
For 2.9 test passed:
image

**Comment by sbulage** [comment]
New 2.10.0 CI Run: https://github.com/rancher/fleet-e2e/actions/runs/11893582861

File: tests/cypress/e2e/unit_tests/p1_fleet.spec.ts [hunk 1]
```diff
@@ -965,6 +971,23 @@ describe("Test Application deployment based on 'clusterGroupSelector'", { tags:
...
+          cy.accesMenuSelection('Continuous Delivery', 'Clusters');
+          cy.contains('.title', 'Clusters').should('be.visible');
+
+          // Add label to the third cluster
+          cy.assignClusterLabel(dsThirdClusterName, key, value);
+
+          // Check application deployed to third cluster
+          cy.wait(500);
+          cy.checkApplicationStatus(appName, dsThirdClusterName, 'All Namespaces');
+
```
Comment (mmartin24) [inline tests/cypress/e2e/unit_tests/p1_fleet.spec.ts:985]: Sugestion for this or other pr: maybe add negative assertion not being deployed on other clusters

File: tests/cypress/e2e/unit_tests/p1_fleet.spec.ts [hunk 2]
```diff
@@ -916,6 +916,12 @@ describe("Test Application deployment based on 'clusterGroupSelector'", { tags:
       test_explanation: "multiple-apps",
       bundle_count: '2 / 2',
     },
+    {
+      qase_id: 32,
+      app: 'single-app',
+      test_explanation: "single-app to the third cluster",
```
Comment (mmartin24) [inline tests/cypress/e2e/unit_tests/p1_fleet.spec.ts:922]: Suggestion:

Maybe add Fleet number at the beginning of test
Rethink second `to` (maybe as above)

image

Comment (sbulage) [reply tests/cypress/e2e/unit_tests/p1_fleet.spec.ts]: Updated test names and added the Fleet-test no. in the beginning.

File: tests/cypress/e2e/unit_tests/p1_fleet.spec.ts (hunk 1 above)
Comment (sbulage) [reply tests/cypress/e2e/unit_tests/p1_fleet.spec.ts:990]: Updated checkApplicationStatus with present: true by default to check application present. Changing present option to false will check it's absence/removal.

I have added such a check where is applicable.

**Comment by sbulage** [comment]
Thanks for the suggestions. :smile:

**Comment by sbulage** [comment]
Another 2.10.0 CI run based on new changes: https://github.com/rancher/fleet-e2e/actions/runs/11911313294

File: tests/cypress/e2e/unit_tests/p1_fleet.spec.ts (hunk 1 above)
Comment (mmartin24) [reply tests/cypress/e2e/unit_tests/p1_fleet.spec.ts:990]: Sorry, not sure I see where the negative check is done on other cluster. Maybe for another pr?

Comment (sbulage) [reply tests/cypress/e2e/unit_tests/p1_fleet.spec.ts:990]: Here I have added it.
https://github.com/rancher/fleet-e2e/pull/236/files#diff-41e30f824d50a402bda9f8b58bf8562c3e9f3ac6fbb1fb193266573fb60a9a45R974-R976

Comment (mmartin24) [reply tests/cypress/e2e/unit_tests/p1_fleet.spec.ts:990]: that it is not what I meant but pls dont want to delay this anymore. 
feel free to merge

**Pull request merged**